src/controllers/userController.js

```diff
@@ -56,6 +58,47 @@ class Authentication {
       // console.log(err);
     }
   }
+
+  /**
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} return an object containing the confirmation message
+   */
+  static async sendEmail(req, res) {
```

Comment:
the static method `sendEmail ` has the same name as the imported `sendEmail ` this will create confusion

Reply:
Done but not yet pushed